plugins/optimization-detective/load.php

```diff
@@ -38,10 +38,9 @@
 require_once __DIR__ . '/storage/class-od-url-metrics-post-type.php';
 require_once __DIR__ . '/storage/data.php';
 require_once __DIR__ . '/storage/rest-api.php';
+OD_URL_Metrics_Post_Type::add_hooks();
```

Comment:
Curious: Why not move it below all the `require_once`s like you had it before? Feels a bit odd to have "logic" mixed up with loading files, but that may be a personal preference.

Reply:
Or actually: Why not put this into `hooks.php`? I'd argue that file should probably be loaded last and add all the hooks, similar to how `default-filters.php` mostly does in WP core.

Reply:
Alternatively, this call could be in `storage.php` which gets required here, but it would be the only code in the file. Down further, `optimization.php` adds a hook as well:

https://github.com/WordPress/performance/blob/18d36fea4d1db730c9443ff776e1f5c0b68d5013/plugins/optimization-detective/optimization.php#L29

Should these two hooks be moved to `load.php` so that none of the requires have any side effects?

Reply:
I had put it here because it goes with the other storage logic.

Reply:
Oh, `hooks.php`, right 😊

Reply:
Done in b74f7ebe180ec7bbe1062cfd57894970d9d21351